.clang-format

```diff
@@ -13,9 +13,15 @@ QualifierAlignment: Right
 
 IncludeBlocks: Merge
 IncludeCategories:
-# Matches common headers first, but sorts them after project includes
 - Regex: '^\"google/cloud/internal/disable_deprecation_warnings.inc\"$'
   Priority: -1
+# System and C-language headers should go last. These expressions may miss a few
+# cases, we can always add more.  Or we can conservative
+- Regex: '^<[A-Za-z0-9_]*\.h>$'
+  Priority: 10000
+- Regex: '^<sys/[A-Za-z0-9_]*\.h>$'
+  Priority: 10000
```

Comment:
Do we expect it to have no effect?

Reply:
Or will it only in the upcoming merge?

Reply:
I expected no change (after removing all the `sys/types.h` includes).  It will have some effect on #13785